src/FixedDataTableCell.js

```diff
@@ -277,11 +260,26 @@ class FixedDataTableCell extends React.Component {
       );
     }
 
-    var cellProps = {
-      columnKey,
-      height,
-      width
-    };
+    var cellProps;
+    if(!this.props.isHeader) {
+      cellProps = {
+        columnKey,
+        height,
+        width,
+      }
+    }
+    else {
```

Comment:
`cellProps` for header cells.

Reply:
I think it might be cleaner to do something like:
```js
const cellProps = {
  columnKey,
  height,
  width, 
};

if (this.props.isHeader) {
  _.extend(cellProps, {
    <restoftheprops>
  });
}
```